packages/api-extractor-model/README.md

```diff
@@ -9,7 +9,7 @@ API documentation for this package: https://rushstack.io/pages/api/api-extractor
 ## Example Usage
 
 The following code sample shows how to load `example.api.json`, which would be generated by API Extractor
-when it analyzes a hypothetical NPM package called `example`:
+when it analyzes a hypothetical NPM package called `example`. The example uses [ES modules](https://nodejs.org/api/esm.html#enabling).
```

Comment:
Don't edit this readme... this comes from upstream and nobody can use it outside of the monorepo anyway...